### ISSUE TITLE
how many step should i set to go

### ISSUE THREAD

**Issue by shartoo** [keithito/tacotron#209]
I have trained tacotron on mandarin corpus for 146k steps now, and the loss seems stable now， but i can't synthesis good voice till now.What should i do ？
![total_loss](https://user-images.githubusercontent.com/5716282/45080438-7df74c80-b127-11e8-9f4e-bd25cfc0fde1.png)

![qq 20180905161800](https://user-images.githubusercontent.com/5716282/45080357-53a58f00-b127-11e8-89c0-c8106ee207ec.png)
![qq 20180905161807](https://user-images.githubusercontent.com/5716282/45080358-53a58f00-b127-11e8-9a32-bbc407fb1037.png)

**Comment by shartoo** [comment]
I am trying to synthesize Mandarin voice consist of  english alphabet A-Z ,number 0-9 and few Chinese word(less than 5),with 3400 wav records.The synthesized voice sounds good at begin and end,while collapse in the middle.After one night training,the loss image looks below,seemed no improvement at all

![linear_loss](https://user-images.githubusercontent.com/5716282/45130669-28b94a80-b1bc-11e8-80db-392c7d8f7fd2.png)
![mel_loss](https://user-images.githubusercontent.com/5716282/45130670-28b94a80-b1bc-11e8-908f-beb69abd729a.png)
![total_loss](https://user-images.githubusercontent.com/5716282/45130672-2951e100-b1bc-11e8-9087-92e0c220322b.png)

**Comment by shartoo** [comment]
All right,just need more training steps,thank you!
